synnax/pkg/distribution/channel/verification/service.go

```diff
@@ -83,14 +83,24 @@ func OpenService(toOpen string, cfgs ...Config) (*Service, error) {
 			return service, nil
 		}
 		service.Ins.L.Info(decode("dXNpbmcgdGhlIGxhc3QgbGljZW5zZSBrZXkgc3RvcmVkIGluIHRoZSBkYXRhYmFzZQ=="))
-		sCtx.Go(service.logTheDog)
+		sCtx.Go(
+			service.logTheDog,
+			signal.WithRetryOnPanic(),
+			signal.WithBaseRetryInterval(2*time.Second),
+			signal.WithRetryScale(1.1),
+		)
 		return service, nil
 	}
 	err = service.create(ctx, toOpen)
 	if err != nil {
 		return service, err
 	}
-	sCtx.Go(service.logTheDog)
+	sCtx.Go(
+		service.logTheDog,
```

Comment:
can put a comment in service.go as to what this is as its not immediately clear from the name